package.json

```diff
@@ -23,14 +23,17 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "npm run lint && mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test": "npm run lint && npm run test-node && npm run test-electron",
+    "test-node": "mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test-electron": "electron-mocha --recursive tests/*.spec.js -R spec",
     "lint": "eslint lib tests bin example.js",
     "readme": "jsdoc2md --template doc/README.hbs lib/index.js > README.md"
   },
   "author": "Juan Cruz Viotti <juan@resin.io>",
   "license": "Apache-2.0",
   "devDependencies": {
     "drivelist": "^5.0.6",
```

Comment:
Not related to this PR specifically, but would it make sense for this `drivelist` version to be updated?